<!DOCTYPE html>
<html lang="en-US">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <meta name="generator" content="VuePress 2.0.0-beta.35">
    <style>
      :root {
        --c-bg: #fff;
      }
      html.dark {
        --c-bg: #22272e;
      }
      html, body {
        background-color: var(--c-bg);
      }
    </style>
    <script>
      const userMode = localStorage.getItem('vuepress-color-scheme');
			const systemDarkMode = window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches;
			if (userMode === 'dark' || (userMode !== 'light' && systemDarkMode)) {
				document.documentElement.classList.toggle('dark', true);
			}
    </script>
    <script src="/mathjax-config.js" defer></script><script id="MathJax-script" type="text/javascript" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-svg.js" defer></script><title>Interaction Environments</title><meta name="description" content="Software Engineer and UI/UX Design Consultant">
    <link rel="modulepreload" href="/assets/app.2c8fd077.js"><link rel="modulepreload" href="/assets/index.html.fbe4bbb9.js"><link rel="modulepreload" href="/assets/index.html.4c38e1fd.js"><link rel="modulepreload" href="/assets/plugin-vue_export-helper.21dcd24c.js">
    <link rel="stylesheet" href="/assets/style.0371b46c.css">
  </head>
  <body>
    <div id="app"><!--[--><div class="theme-container no-sidebar"><!--[--><header ref_key="navbar" class="navbar"><div class="toggle-sidebar-button" title="toggle sidebar" aria-expanded="false" role="button" tabindex="0"><div class="icon" aria-hidden="true"><span></span><span></span><span></span></div></div><span><a href="/" class=""><!----><span class="site-name can-hide">Interaction Environments</span></a></span><div class="navbar-items-wrapper" style=""><!--[--><!--]--><nav class="navbar-items can-hide"><!--[--><div class="navbar-item"><a aria-current="page" href="/portfolio/" class="router-link-active router-link-exact-active router-link-active" aria-label="Portfolio"><!--[--><!--]--> Portfolio <!--[--><!--]--></a></div><div class="navbar-item"><a class="external-link" href="https://github.com/filonik" rel="noopener noreferrer" target="_blank" aria-label="GitHub"><!--[--><!--]--> GitHub <span><svg class="external-link-icon" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false" x="0px" y="0px" viewbox="0 0 100 100" width="15" height="15"><path fill="currentColor" d="M18.8,85.1h56l0,0c2.2,0,4-1.8,4-4v-32h-8v28h-48v-48h28v-8h-32l0,0c-2.2,0-4,1.8-4,4v56C14.8,83.3,16.6,85.1,18.8,85.1z"></path><polygon fill="currentColor" points="45.7,48.7 51.3,54.3 77.2,28.5 77.2,37.2 85.2,37.2 85.2,14.9 62.8,14.9 62.8,22.9 71.5,22.9"></polygon></svg><span class="external-link-icon-sr-only">open in new window</span></span><!--[--><!--]--></a></div><!--]--></nav><!--[--><!--]--><button class="toggle-dark-button" title="toggle dark mode"><svg style="" class="icon" focusable="false" viewBox="0 0 32 32"><path d="M16 12.005a4 4 0 1 1-4 4a4.005 4.005 0 0 1 4-4m0-2a6 6 0 1 0 6 6a6 6 0 0 0-6-6z" fill="currentColor"></path><path d="M5.394 6.813l1.414-1.415l3.506 3.506L8.9 10.318z" fill="currentColor"></path><path d="M2 15.005h5v2H2z" fill="currentColor"></path><path d="M5.394 25.197L8.9 21.691l1.414 1.415l-3.506 3.505z" fill="currentColor"></path><path d="M15 25.005h2v5h-2z" fill="currentColor"></path><path d="M21.687 23.106l1.414-1.415l3.506 3.506l-1.414 1.414z" fill="currentColor"></path><path d="M25 15.005h5v2h-5z" fill="currentColor"></path><path d="M21.687 8.904l3.506-3.506l1.414 1.415l-3.506 3.505z" fill="currentColor"></path><path d="M15 2.005h2v5h-2z" fill="currentColor"></path></svg><svg style="display:none;" class="icon" focusable="false" viewBox="0 0 32 32"><path d="M13.502 5.414a15.075 15.075 0 0 0 11.594 18.194a11.113 11.113 0 0 1-7.975 3.39c-.138 0-.278.005-.418 0a11.094 11.094 0 0 1-3.2-21.584M14.98 3a1.002 1.002 0 0 0-.175.016a13.096 13.096 0 0 0 1.825 25.981c.164.006.328 0 .49 0a13.072 13.072 0 0 0 10.703-5.555a1.01 1.01 0 0 0-.783-1.565A13.08 13.08 0 0 1 15.89 4.38A1.015 1.015 0 0 0 14.98 3z" fill="currentColor"></path></svg></button><!----></div></header><!--]--><div class="sidebar-mask"></div><!--[--><aside class="sidebar"><nav class="navbar-items"><!--[--><div class="navbar-item"><a aria-current="page" href="/portfolio/" class="router-link-active router-link-exact-active router-link-active" aria-label="Portfolio"><!--[--><!--]--> Portfolio <!--[--><!--]--></a></div><div class="navbar-item"><a class="external-link" href="https://github.com/filonik" rel="noopener noreferrer" target="_blank" aria-label="GitHub"><!--[--><!--]--> GitHub <span><svg class="external-link-icon" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false" x="0px" y="0px" viewbox="0 0 100 100" width="15" height="15"><path fill="currentColor" d="M18.8,85.1h56l0,0c2.2,0,4-1.8,4-4v-32h-8v28h-48v-48h28v-8h-32l0,0c-2.2,0-4,1.8-4,4v56C14.8,83.3,16.6,85.1,18.8,85.1z"></path><polygon fill="currentColor" points="45.7,48.7 51.3,54.3 77.2,28.5 77.2,37.2 85.2,37.2 85.2,14.9 62.8,14.9 62.8,22.9 71.5,22.9"></polygon></svg><span class="external-link-icon-sr-only">open in new window</span></span><!--[--><!--]--></a></div><!--]--></nav><!--[--><!--]--><!----><!--[--><!--]--></aside><!--]--><!--[--><main class="page"><!--[--><!--]--><div class="theme-default-content"><!--[--><h2 id="projects" tabindex="-1"><a class="header-anchor" href="#projects" aria-hidden="true">#</a> Projects</h2><h3 id="datachopin" tabindex="-1"><a class="header-anchor" href="#datachopin" aria-hidden="true">#</a> DataChopin</h3><p><img class="object-cover h-80 w-full shadow-lg shadow-black/50" src="/images/projects/datachopin/screenshot-1-full-res.png"></p><p>DataChopin a novel interface for exploratory data analysis in collaborative interaction environments. Specifically, this refers to collaborative spaces such as meeting rooms or research labs, which are instrumented with interactive technologies. The distinctive features of DataChopin are the use of a large-scale display wall as a shared desktop, as well as flexible composition mechanisms for incremental construction of visualisations. The available display space supports multiple, simultaneous users, who are free to roam in front of the display and contribute to the analysis at any time.</p><h3 id="massive-networks" tabindex="-1"><a class="header-anchor" href="#massive-networks" aria-hidden="true">#</a> Massive Networks</h3><p><img class="object-cover h-80 w-full shadow-lg shadow-black/50" src="/images/projects/massive-networks/photo-3.jpg"></p><p>Massive Networks is a project that explores the use of high-end visualisation systems for interactive analysis of very large and complex networks. The interface suports a variety of common analytics tasks, such as selection, filtering, layout changes, and summarisation. Furthermore, it was recently extended to support graph visualisaiton using non-euclidean geometry. As such, it is well suited for high-end visualisation systems, which come in a wide variety of display sizes and shapes.</p><h3 id="genomics" tabindex="-1"><a class="header-anchor" href="#genomics" aria-hidden="true">#</a> Genomics</h3><p><img class="object-cover h-80 w-full shadow-lg shadow-black/50" src="/images/projects/genomics/photo-1.png"></p><p>The genomics project visualises gene expressions accross the entire human genome. It utilises a unique dataset compiled by researchers at UNSW&#39;s Children&#39;s Cancer Institute that allows to observe changes in gene expressions over time. The resulting amount of data is very large and overwhelms conventional genomics tools. Consequently, it is a great example of data analytics that requires the capabilities of high-end visualisation systems.</p><h3 id="biodive" tabindex="-1"><a class="header-anchor" href="#biodive" aria-hidden="true">#</a> BioDIVE</h3><p><img class="object-cover h-80 w-full shadow-lg shadow-black/50" src="/images/projects/biodive/photo-2.jpg"></p><p>BioDIVE is a research prototype developed to visualise changes in cell morphology. It comprises a set of bespoke visualisations that integrate bio-images and derived quantitative data, enabling deeper comprehension and seamless interaction with multi-dimensional cellular information. The multivariate dataset contains 52 microscopy image-derived parameters. This information is visualised in an immersive 3d environment through several dimension reduction techniques.</p><h2 id="publications" tabindex="-1"><a class="header-anchor" href="#publications" aria-hidden="true">#</a> Publications</h2><p>You can find a complete listing of my academic publications by visiting my profile <a href="https://scholar.google.com/citations?user=qmOacP8AAAAJ" target="_blank" rel="noopener noreferrer">on Google Scholar<span><svg class="external-link-icon" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false" x="0px" y="0px" viewbox="0 0 100 100" width="15" height="15"><path fill="currentColor" d="M18.8,85.1h56l0,0c2.2,0,4-1.8,4-4v-32h-8v28h-48v-48h28v-8h-32l0,0c-2.2,0-4,1.8-4,4v56C14.8,83.3,16.6,85.1,18.8,85.1z"></path><polygon fill="currentColor" points="45.7,48.7 51.3,54.3 77.2,28.5 77.2,37.2 85.2,37.2 85.2,14.9 62.8,14.9 62.8,22.9 71.5,22.9"></polygon></svg><span class="external-link-icon-sr-only">open in new window</span></span></a>.</p><!--]--></div><footer class="page-meta"><!----><!----><!----></footer><!----><!--[--><!--[--><div class="home py-0"><div class="footer">Â© Copyright Daniel Filonik 2022. All rights reserved.</div></div><!--]--><!--]--></main><!--]--></div><!----><!--]--></div>
    <script type="module" src="/assets/app.2c8fd077.js" defer></script>
  </body>
</html>
